cpp/src/barretenberg/honk/sumcheck/relations/auxiliary_relation.hpp

```diff
@@ -4,26 +4,37 @@
 
 #include "../polynomials/univariate.hpp"
 #include "barretenberg/numeric/uint256/uint256.hpp"
-#include "relation.hpp"
+#include "relation_parameters.hpp"
 
 namespace proof_system::honk::sumcheck {
 
 template <typename FF> class AuxiliaryRelation {
   public:
     // 1 + polynomial degree of this relation
-    static constexpr size_t RELATION_LENGTH = 6; // degree() = WORKTODO
+    static constexpr size_t RELATION_LENGTH = 6;
+
+    static constexpr size_t NUM_CONSTRAINTS = 6;
+    static constexpr std::array<size_t, NUM_CONSTRAINTS> CONSTRAINT_LENGTH = { 6, 6, 6, 6, 6, 6 }; // Each has degree 5
```

Comment:
It just so happens that all constraints for this relation have the same length. (I think it turned out this way because Zac was trying to squeeze as much functionality out of each one as possible while not increasing the degree of the overall Sumcheck relation). If you look at say the elliptic or permutation relations you'll see that they dont in general all have the same degree.